gpu/cuda_common.go

```diff
@@ -19,3 +27,38 @@ func cudaGetVisibleDevicesEnv(gpuInfo []GpuInfo) (string, string) {
 	}
 	return "CUDA_VISIBLE_DEVICES", strings.Join(ids, ",")
 }
+
+func cudaGetVariant(gpuInfo CudaGPUInfo) string {
+	if runtime.GOARCH == "arm64" && runtime.GOOS == "linux" {
```

Comment:
This seems like its should potentially be a dedicated arm64 function

Reply:
My preference would be to leave it as we do need to fall through to the computeMajor and DriverMinor comparison on arm64 SBSA systems, so splitting this into a `cuda_linux_arm64.go` doesn't actually simplify it down as cleanly.